bats/foreman_helper.bash

```diff
@@ -46,10 +46,10 @@ tSkipIfOlderThan43() {
   fi
 }
 
-tSkipIfNewerThan45() {
```

Comment:
This is still used in https://github.com/theforeman/forklift/blob/d8db449a566e41b0476b1bbfdb737f48af611e32/bats/fb-katello-content.bats#L255-L256

However, we don't really have to care, as 4.5+ is the only thing we care about these days, so we can just delete the whole "add subscription to activation key" test and then also delete this `SkipIfNewer` definition.